<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ex_1_2_003Class</class>
 <widget class="QMainWindow" name="Ex_1_2_003Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ex_1_2_003</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Btn_IC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>计算相交和包含数</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_Intersacted">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>相交数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lable_Contains">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>包含数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_N">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Min">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Ex_1_2_003.qrc"/>
 </resources>
 <connections/>
</ui>
